src/v/cluster/controller_backend.cc

```diff
@@ -370,7 +372,7 @@ controller_backend::bootstrap_ntp(const model::ntp& ntp, deltas_t& deltas) {
     auto bootstrap_deltas = calculate_bootstrap_deltas(_self, deltas);
     vlog(
       clusterlog.trace,
-      "bootstrapping {} with deltas {}",
+      "[{}] bootstrapping with deltas {}",
```

Comment:
nit: probably not too important yet, but it might be worthwhile to have some `vlog_with_ntp()` method that takes as input a `model::ntp` so we can reuse it in places, and so it's hard to mess up ordering.

Reply:
good idea, will think how to make it neatly in a follow up PR. The `vlog` is actually a macro to have the least performance overhead